### ISSUE TITLE
shutdown hijacking the system shutdown command

### ISSUE THREAD

**Issue by fxn** [brandonhilkert/sucker_punch#166]
Looks like installing sucker_punch hijacks the system `shutdown` command:

```
vagrant@rails-dev-box:~$ sudo shutdown -h now
/usr/lib/ruby/vendor_ruby/bundler.rb:198:in `rescue in root': Could not locate Gemfile or .bundle/ directory (Bundler::GemfileNotFound)
	from /usr/lib/ruby/vendor_ruby/bundler.rb:194:in `root'
	from /usr/lib/ruby/vendor_ruby/bundler.rb:106:in `bundle_path'
	from /usr/lib/ruby/vendor_ruby/bundler.rb:439:in `configure_gem_home'
	from /usr/lib/ruby/vendor_ruby/bundler.rb:431:in `configure_gem_home_and_path'
	from /usr/lib/ruby/vendor_ruby/bundler.rb:92:in `configure'
	from /usr/lib/ruby/vendor_ruby/bundler.rb:153:in `definition'
	from /usr/lib/ruby/vendor_ruby/bundler.rb:123:in `setup'
	from /usr/lib/ruby/vendor_ruby/bundler.rb:134:in `require'
	from /var/lib/gems/2.3.0/gems/sucker_punch-2.0.0/bin/shutdown:4:in `<top (required)>'
	from /usr/local/bin/shutdown:23:in `load'
	from /usr/local/bin/shutdown:23:in `<main>'
```

Maybe change the commands' name?

**Comment by brandonhilkert** [comment]
Released `2.0.1`

**Comment by fxn** [comment]
That was fast :heart:.
